qcfractal/interface/models/gridoptimization.py

```diff
@@ -299,11 +303,12 @@ def get_scan_dimensions(self) -> Tuple[List[float]]:
         for scan in self.keywords.scans:
             ret.append(len(scan.steps))
 
-        return tuple(ret)
+        ret = (ret, )
```

Comment:
I changed this return to match the Typing hint of the call, but now it causes CI to fail due to:

```
FractalServer Service Build and Iterate Error:
Traceback (most recent call last):
  File "/Users/levinaden/github/qcfractal/qcfractal/server.py", line 480, in update_services
    finished = service.iterate()
  File "/Users/levinaden/github/qcfractal/qcfractal/services/gridoptimization_service.py", line 177, in iterate
    new_points_list = expand_ndimensional_grid(self.dimensions, self.seeds, self.complete)
  File "/Users/levinaden/github/qcfractal/qcfractal/services/service_util.py", line 201, in expand_ndimensional_grid
    if new_dim >= dimensions[d]:
TypeError: '>=' not supported between instances of 'int' and 'list'
```

Which happens because the return is now a `Tuple[List[float]]` instead of `Tuple[float, ...]` which is what it was. It appears the correct call should be `Tuple[float, ...]`, but I am not sure if the fix should be revert my change and adjust the return Type hint, or change the way the dimensions are checked outside of here.

Reply:
Should be `Tuple[float, ...]`. As we do not correctly validate type hints the current return is always the correct solution.

Reply:
Great, I'll fix that